Entity/AbstractCrudEntity.php

```diff
@@ -20,7 +23,7 @@
  *
  * @author Sergey Chernecov <sergey.chernecov@intexsys.lv>
  */
-abstract class AbstractCrudEntity implements CrudEntityInterface
+abstract class AbstractCrudEntity implements CrudEntityInterface, EmbeddedInterface, TransactionalInterface, TimestampableInterface
```

Comment:
I've only removed timestampable because individuals may not need this specific updatedAt and createdAt behavior on all of their CRUD entities. I will create an issue to better deal with the transactional and embedded interfaces.